no-literal-qual/src/main/java/org/checkerframework/checker/noliteral/qual/PolyConstant.java

```diff
@@ -0,0 +1,15 @@
+package org.checkerframework.checker.noliteral.qual;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.PolymorphicQualifier;
+
+/** A polymorphic qualifier for the no literal type system. */
```

Comment:
This checker doesn't have a manual section (because it's not in the Checker Framework's source tree), so I clearly cannot link to it.

I will add a link to the section about polymorphic qualifiers.

Reply:
It does have a manual at https://github.com/kelloggm/no-literal-checker, and it is worth linking to the manual.

Reply:
Ah I misunderstood your comment to be asking for a link to the Checker Framework manual specifically. I've added links to the `README.md` file.